modules/apps/portal/portal-error-code-test/src/testIntegration/java/com/liferay/portal/error/code/test/PortalErrorCodeTest.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.error.code.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.util.URLUtil;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+
+import java.net.URL;
+
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author István András Dézsi
+ */
+@RunWith(Arquillian.class)
+public class PortalErrorCodeTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new LiferayIntegrationTestRule();
+
+	@Test
+	public void testAccessErrorsCodeJsp() throws Exception {
+		URLUtil.toString(new URL("http://localhost:8080/errors/code.jsp"));
+	}
+
+}
```

Comment:
@IstvanD, the current test doesn't exist Assert. After the test finished, the portal existed the WARN log `2023-12-27 08:46:05.747 WARN  [http-nio-8080-exec-6][code_jsp:157] {code="0", msg="null", uri=null}`,  I remember ci test will fail the test if it found WARN log existed in the log. So please use `LoggerTestUtil.configureLog4JLogger` to do the Assert. The `code_jsp` loggerName actually is `portal_web.docroot.errors.code_jsp`, so you may use the below codes to assert.
```
try (LogCapture logCapture = LoggerTestUtil.configureLog4JLogger("portal_web.docroot.errors.code_jsp", LoggerTestUtil.WARN)) {
			URLUtil.toString(new URL("http://localhost:8080/errors/code.jsp"));

			List<LogEntry> logEntries = logCapture.getLogEntries();

			Assert.assertEquals(logEntries.toString(), 1, logEntries.size());

			LogEntry logEntry = logEntries.get(0);

			Assert.assertEquals(LoggerTestUtil.WARN, logEntry.getPriority());
			
			String message = logEntry.getMessage();

			Assert.assertEquals(
				"{code=\"0\", msg=\"null\", uri=null}", logEntry.getMessage());
		}
```